README.md

```diff
@@ -8,19 +8,78 @@
-* Run an R script against the exported tables, producing one or more report artefacts
-* Save a set of metadata
-* Archive the exported data set
+The user describes the inputs of their analysis, including:
+
+* SQL queries (if using databases)
+* Required R sources
+* External resource files (e.g., csv data files, Rmd files, templates)
+* Packages required to run the analysis
+* Dependencies on previously run analyses
+
+Along with a list of "artefacts" (file-based results) that they will produce.
+
+Then orderly:
+
+1. creates a new empty directory
+2. copies over _only_ the declared file resources
+3. loads only the declared packages
+4. loads the declared R sources
+5. evaluates any sql queries to create R objects
+6. then runs then analysis
+7. verifies that the declared artefacts are produced
+
+It then stores metadata alongside the analysis including md5 hashes of all inputs and outputs, copies of data extracted from the database, a record of all R packages loaded at the end of the session, and (if using git) information about the git state (hash, branch and status).
+
+## Workflows with `orderly`
+
+In the [MRC Centre for Global Infectious Disease Analysis](https://www.imperial.ac.uk/mrc-global-infectious-disease-analysis) we use orderly on two major projects.
+
+- The [Vaccine Impact Modelling Consortium](https://www.vaccineimpact.org/)
+- Our part of the response to the 2018-2019 ebola outbreak in DRC
+
+The workflows we have developed here are oriented towards collaborative groups of researchers - other workflows are possible (indeed orderly is also designed to support a _decentralised_ workflow, though this has not been used in practice yet).
+
+In these projects we have a group of researchers who develop and test analyses locally.  These are developed on a [branch in git](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) and then run on a centralised _staging environment_ (a duplicate of our production environment).  The code and outputs are reviewed with the help of [GitHub's "Pull requests"](https://help.github.com/en/articles/about-pull-requests) and then the reports are run on our production environment.
+
+Interaction with the remote environments is achieved using an [HTTP API](https://github.com/vimc/montagu-reporting-api) which orderly itself transparently uses, so that running reports remotely can be run [directly from R](https://vimc.github.io/orderly/reference/orderly_run_remote.html).  The remote systems also an interactive web interface that can be used to explore and download versions of analyses, as well as run new ones.
 
-## Database schema
+## Internal database schema
 
 `orderly` has a database, which should be the preferred way of querying `orderly` from other programs.  The schema is programmatically described at [`inst/database/schema.yml`](inst/database/schema.yml) and automatically generated database documentation is available [here](https://vimc.github.io/orderly/schema).
```

Comment:
Perhaps below, edit one of the `orderly`s -> The preferred way of querying `orderly` is via its own database.